crypto/ct/ct_sct.c

```diff
@@ -223,6 +237,54 @@ int SCT_set1_signature(SCT *sct, const unsigned char *sig, size_t sig_len)
     return 1;
 }
 
+int SCT_set1_serialized_signature(SCT *sct, const unsigned char **in,
```

Comment:
Agreed, it really should be something like TLS_DIGITALLY_SIGNED and live in the TLS code, but that seemed even less likely to get reviewed in time. I'm quite happy not to rush in something like the CT_SIGNATURE change, but I thought I'd give you the option since I already had the code lying around.